frontend/src/pages/Warning.jsx

```diff
@@ -1,3 +1,5 @@
+import '../lib/ErrorReporting'
```

Comment:
Anyway that this boilerplate can be avoided.

Reply:
This was the shortest I could make it, unless @hueyy has any ideas. From my understand each page is an entry point so we need it on each of them.

Reply:
Yeah the way it's set up right now requires this. Not worth doing something fancy like injecting it into the React dependency via webpack imo.